File: src/main/java/no/novari/flyt/instance/Application.java
package no.novari.flyt.instance;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication(scanBasePackages = {
        "no.novari.flyt.instance",
        "no.fintlabs"
})
@EnableScheduling
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: src/main/kotlin/no/novari/Application.kt
package no.novari

import no.novari.flyt.audit.config.EnableFlytAuditing
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
@EnableFlytAuditing
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
